driver/main_args.ml

```diff
@@ -179,6 +179,12 @@ let mk_dump_pass f =
     !Clflags.all_passes
 ;;
 
+let mk_oclassic f =
+  "-Oclassic", Arg.Unit f, " Make inlining decisions at function definition \
```

Comment:
I would replace this help text to say " Emulate optimisation of non-Flambda compilers"

Reply:
done